Content.Server/Shuttles/Systems/ShuttleSystem.EmergencyShuttle.cs

```diff
@@ -256,8 +256,16 @@ public void CallEmergencyShuttle(EntityUid? stationUid)
 
        if (TryHyperspaceDock(shuttle, targetGrid.Value))
        {
+           if (TryComp<TransformComponent>(targetGrid.Value, out var targetXform))
+           {
+               var mapDiff = xform.WorldPosition - targetXform.WorldPosition;
+               var targetRotation = targetXform.WorldRotation;
+               var angle = mapDiff.ToWorldAngle();
+               angle -= targetRotation;
```

Comment:
Wondering if I should consider centre of mass because I feel the mappers may not always have 0,0 as the rough origin.